src/main/java/com/google/sps/servlets/PlaceGuideServlet.java

```diff
@@ -0,0 +1,111 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.data.RepositoryType;
+import com.google.sps.placeGuide.PlaceGuide;
+import com.google.sps.placeGuide.repository.PlaceGuideRepository;
+import com.google.sps.placeGuide.repository.PlaceGuideRepositoryFactory;
+import com.google.appengine.api.datastore.GeoPt;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import java.lang.IllegalStateException;
+import java.io.IOException;
+
+/**
+ * This servlet handles placeguide's data.
+ */
+@WebServlet("/place-guide-data")
+public class PlaceGuideServlet extends HttpServlet {
+  public static final String ALL = "all";
+  public static final String CREATED_ALL = "created-all";
+  public static final String CREATED_PUBLIC = "created-public";
+  public static final String CREATED_PRIVATE = "created-private";
```

Comment:
Maybe use an enum for these query types to make the code cleaner.

Reply:
done.

Reply:
done.